### DISCUSSION TITLE
Image for linux/arm64/v8

### DISCUSSION THREAD

**Discussion by erikmiers** [wolveix/satisfactory-server#190] [General]
Hi, I would like to run this on an arm64 linux.

I tried building it from scratch with docker build, but it didnt work.
Since I now nothing about docker - I have no idea how to fix the build.

Any ideas how to do this?

Thanks,
 Erik

**Comment by wolveix** [comment]
Hey @erikmiers, this is a duplicate of #174. Satisfactory's server doesn't support ARM, so there's nothing we can do unfortunately.
